src/webview/html/messageHeaderAsHtml.js

```diff
@@ -87,11 +88,7 @@ export default (
   }
 
   if (item.type === 'private' && headerStyle === 'full') {
-    const recipients =
-      item.display_recipient.length === 1 && item.display_recipient[0].email === ownUser.email
-        ? item.display_recipient
-        : item.display_recipient.filter(r => r.email !== ownUser.email);
-
+    const recipients = filteredRecipientsForPM(item, ownUser);
     const narrowObj =
       recipients.length === 1
```

Comment:
`recipients` is being used here too, before map, sort aand join. If I include map, sort, and join in the common function I created, I won't be able to reuse it here. Do you still want me to include map, sort, and join in the new function I created?

Reply:
Sorry, I'd missed that on first readthrough – and unfortunately, that value seems to flow into a morass of insufficiently-undocumented assumptions.

Ah well. Perhaps in some future version.